### ISSUE TITLE
Not a Issue but a question.

### ISSUE THREAD

**Issue by Therealjosephchrzempiec** [ElectronicsTinkerer/1541EmU#2]
Hello i was looking at your code and I was wondering what is the ESP8266 for? Sorry  a little lost to see that in the code.

Joseph

**Comment by ElectronicsTinkerer** [comment]
Hello,
Sorry about the slow response, I am pretty new to GitHub and did not realize that there was the "Issues" tab. Anyway, that's a good question. I originally had planned to add an ESP 8266 to the board later on to give the board the ability to connect over WiFi. In v3.25 of the code, I still had the comments (and some code) indicating that the ESP 8266 was still on board and accessible by addressing device 16. As of right now, EFS v1.00 is the latest version of the code that I have released, so make sure that you are running that one. The older versions (v3.xx and v2.xx) don't have support for long filenames, so I would definitely recommend the newer one (although it is a bit more complex in terms of accessing a file). (I believe) I have removed all of the ESP 8266 code from EFS v1.00, so I wouldn't worry about it.

TL;DR: The ESP 8266 was included in older versions of the code, as I had planned on adding support, however, I have removed it from the latest version to simplify the code. So, it is not needed for it to work.

Hope that helps!

**Comment by SuperNeotendo** [comment]
just an idea what was in my head: what if you compile this code on the ESP32 and add 6502/CIA/VIA Emulation?

**Comment by ElectronicsTinkerer** [comment]
That could be an interesting expansion to the project. It might even allow fast loaders. I do have a instruction-resolution [65816 emulator](https://github.com/ElectronicsTinkerer/816CE) that could be dropped in as long as the software did not require any undocumented opcodes or cycle-specific timings.

**Comment by SuperNeotendo** [comment]
yeah, i found an C64 emulator for ESP32 maybe you can use some code from there, 6502 emulation atleast

**Comment by ElectronicsTinkerer** [comment]
Archiving repository, 2026-04-12
